_blinkr/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.1.10
+MAINTAINER ihamilto@redhat.com
+RUN apt-get update
+RUN apt-get install sudo
+
```

Comment:
Both RUN commands should be done in a single line, and you need to clean up after, all on the same line. This helps keep the docker images small.